go.mod

```diff
@@ -1,36 +1,36 @@
 module github.com/openshift/node-observability-operator
 
-go 1.17
+go 1.19
 
 require (
 	github.com/coreos/ignition/v2 v2.13.0
-	github.com/go-logr/logr v1.2.2
+	github.com/go-logr/logr v1.2.3
 	github.com/golangci/golangci-lint v1.42.1
```

Comment:
This needs to be updated to the latest version, to prevent the OOM we saw in release repository.

Reply:
Bumping to the latest version 1.50.0